spinc/cmd/start.go

```diff
@@ -188,16 +189,51 @@ func (c *Start) Run() error {
 	}
 
 	fmt.Printf("OK, started %s request %s\n\n"+
-		"  spinc status %s\n\n", c.reqName, reqId, reqId)
+		"  spinc status %s%s\n\n", c.reqName, reqId, c.statusUserOptions(), reqId)
 
 	return nil
 }
 
+func (c *Start) statusUserOptions() string {
+	var userOptions string
+
+	// Add critical options to the output if set
+	if c.ctx.UserOptions.Env != "" {
+		userOptions = "--env " + escapeArg(c.ctx.Options.Env) + " "
+	}
+
+	if c.ctx.UserOptions.Addr != "" {
+		userOptions += "--addr " + escapeArg(c.ctx.Options.Addr) + " "
+	}
+
+	if c.ctx.UserOptions.Config != "" {
+		userOptions += "--config " + escapeArg(c.ctx.Options.Config) + " "
+	}
+
+	return userOptions
+}
+
 func (c *Start) Cmd() string {
 	if c.fullCmd != "" {
 		return c.fullCmd
 	}
-	fullCmd := "start " + c.reqName
+
+	var fullCmd string
+
+	// Add critical options to the output if set
+	if c.ctx.UserOptions.Env != "" {
+		fullCmd = "--env " + escapeArg(c.ctx.Options.Env) + " "
+	}
+
+	if c.ctx.UserOptions.Addr != "" {
+		fullCmd += "--addr " + escapeArg(c.ctx.Options.Addr) + " "
+	}
+
+	if c.ctx.UserOptions.Config != "" {
+		fullCmd += "--config " + escapeArg(c.ctx.Options.Config) + " "
+	}
```

Comment:
Since this is duplicating what statusUserOptions does, you could just call that func